castai/data_source_resource_rebalancing_schedule_test.go

```diff
@@ -50,7 +50,10 @@ func TestRebalancingScheduleDataSourceRead(t *testing.T) {
                     "evictGracefully": false,
                     "aggressiveMode": false,
                     "aggressiveModeConfig": {
-                        "ignoreLocalPersistentVolumes": true	
+                        "ignoreLocalPersistentVolumes": true,
+						"ignoreProblemJobPods": true,
+						"ignoreProblemRemovalDisabledPods": true,
+						"ignoreProblemPodsWithoutController": true
```

Comment:
could you fix indentation here?

Reply:
I’ll make the changes you've suggested in a separate PR, since this one has already been merged. Sounds fair?